esx_service/vmci/vmci_client.c

```diff
@@ -59,7 +61,9 @@ typedef struct be_request {
 #define BIND_RETRY_COUNT (MAX_CLIENT_PORT - START_CLIENT_PORT)
 
 typedef struct be_answer {
-   char *buf;   // calloced, so needs to be free()
+   char *buf;                  // response buffer
+   char errBuf[ERR_BUF_LEN];   // error response buffer
+   char *errBufP;              // error response pointer
```

Comment:
errBuf & errBufP looks very fishy. Why not use same logic as "buf". Allocated in C and freed in go?